cmd/notifier/config/config.toml

```diff
@@ -6,3 +6,5 @@ subscribed_addresses = ["erd1", "erd1"]
     marshaller_type = "gogo protobuf"
     # Retry duration (receive/ send ack signal) in seconds
     retry_duration = 5
+    # Signals if in case of data payload processing error, we should send the ack signal or not
```

Comment:
missing comma? `Signals if, in case of data...`

Reply:
No missing comma I guess